addons/purchase_stock/__manifest__.py

```diff
@@ -21,6 +21,7 @@
         'views/stock_lot_views.xml',
         'views/stock_orderpoint_views.xml',
         'views/product_views.xml',
+        'views/help_message_template.xml',
```

Comment:
I.M.O. This commit introduces a technical debt we don't want to manage. I know we already speak about it but:
- Isn't it possible to delete all the actions, action_picking_tree_ready,late,backorders,...
- Instead use a dynamic filter in the python code.
- Depending the picking type open the related action action_picking_tree_incoming,outgoing,..
- Set the correct help message in the xml of those views

Reply:
As discussed, I will create a task to improve the help message logic and get rid of duplicated actions.